graphql-samples/mutations/update/assign-group-focal-point

```diff
@@ -0,0 +1,13 @@
+mutation assignGroupFocalPoint($userID: Float!, $groupID: Float!) {
+  assignGroupFocalPoint(groupID: $groupID, userID: $userID) {
+    name,
+    id,
+    focalPoint
```

Comment:
You need to provide focalPoint fields, otherwise it changes (I ran the sample and it threw an exception)